data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -263,22 +179,22 @@ def workforce_factor(self) -> bool:
         # (necessary to screen out university block groups)
         workforce_criteria = (
             (
-                self.df[self.UNEMPLOYMENT_FIELD]
+                self.df[FN.UNEMPLOYMENT_FIELD]
                 > self.ENVIRONMENTAL_BURDEN_THRESHOLD
             )
             | (
-                self.df[self.MEDIAN_INCOME_FIELD]
+                self.df[FN.MEDIAN_INCOME_PERCENT_AMI_FIELD]
                 # Note: a high median income as a % of AMI is good, so take 1 minus the threshold to invert it.
                 # and then look for median income lower than that (not greater than).
                 < 1 - self.ENVIRONMENTAL_BURDEN_THRESHOLD
             )
             | (
-                self.df[self.POVERTY_LESS_THAN_100_FPL_FIELD]
+                self.df[FN.POVERTY_LESS_THAN_100_FPL_FIELD]
```

Comment:
Unfortunately it looks like this points to the wrong field -- `POVERTY_LESS_THAN_100_FPL_FIELD` is `"Percent of individuals < 100% Federal Poverty Line"`, but we actually want `"Percent of individuals < 100% Federal Poverty Line (percentile)"`. Because `POVERTY_LESS_THAN_100_FPL_FIELD` is used in some other scores, we should probably create a new field, like `POVERTY_LESS_THAN_100_FPL_FIELD_PERCENTILE`, or maybe more grammatically accurately,  `POVERTY_LESS_THAN_100_FPL_PERCENTILE_FIELD`. 

Most of these variables are going to point to the percentile, and we'll get pretty inaccurate results if we don't fix that up. 

The ones I know we need to change to a new field name that has ` "percentile"` added to it (I haven't checked all of them): 
* MEDIAN_INCOME_PERCENT_AMI_FIELD
* UNEMPLOYMENT_FIELD
* ASTHMA_FIELD (it looks like ASTHMA_FIELD_PERCENTILE already exists in field_names though)
* DIABETES_FIELD (it looks like DIABETES_FIELD_PERCENTILE already exists in field_names though)
* etc

Reply:
ah okay!